bin/atlassian-backup

```diff
@@ -61,7 +67,11 @@ function backup_attachments() {
 function dump_database() {
     echo "  - Dumping $NATURAL_TOOL_NAME database"
     DATABASE_DUMP_OUTPUT="${OUTPUT_PREFIX}-database-dump.sql.gz"
-    $PGENGINE/pg_dump "$DB_NAME" | gzip > "$DATABASE_DUMP_OUTPUT"
+    if [ "${DB_BACKEND}" = 'mysql' ]; then
+        $MYENGINE --single-transaction --host="$MYSQL_HOST" --user="$MYSQL_USER" --password="$MYSQL_PASS" "$DB_NAME" | gzip > "$DATABASE_DUMP_OUTPUT"
+    else
+        $PGENGINE/pg_dump "$DB_NAME" | gzip > "$DATABASE_DUMP_OUTPUT"
+    fi
```

Comment:
else
    echo "ERROR: No such DB_BACKEND '$DB_BACKEND'" 1>&2
    exit 2
fi

Reply:
I've added this in as well.